src/Project.hs

```diff
@@ -104,6 +104,9 @@ data ApprovedFor
   = Merge
   | MergeAndDeploy
   | MergeAndTag
+  | MergeOnFriday
+  | MergeAndDeployOnFriday
+  | MergeAndTagOnFriday
```

Comment:
Design-wise, I'm not sure this is the right way to go, as it results in a combinatorial blow-up when more options are added.

The fact that it's Friday doesn't change anything inherently about the command, a `MergeAndDeploy` will still merge and deploy. What changes is the way how that command is parsed. I think something like our original `ApprovedFor` together with a `Fridayness` type (e.g. `data Fridayness = It'sFriday | It'sNotFriday`) in a `(ApprovedFor, Fridayness)` tuple would reflect more accurately what is going on and avoid the blow-up.